### PR TITLE
EMBPD00174598:: [EB][WPA4 CE6][IE]: Unable to enter Alphabetical characters through hardware keys inside Textbox

### PR DESCRIPTION
**Pull request by qjt376**
Issue is fixed. Please review the same

### CODE REVIEW

File: platform/wm/rhodes/browser/CEBrowserEngine.cpp [hunk 1]
```diff
@@ -916,6 +926,16 @@ void CEBrowserEngine::RunMessageLoop(CMainWindow& mainWnd)
 		
 		if ((msg.message == WM_KEYDOWN || msg.message == WM_KEYUP) && msg.wParam != VK_BACK)	//  Run Browser TranslateAccelerator
 		{
+			//Setting foucs back to the editable window if the focus is lost
+			if( _tcsicmp(getEditableWndClassName,TEXT("Internet Explorer_Server")) ==0 )
+			{
+				HWND checkFocusWnd = GetFocus();
+				if((checkFocusWnd != getEditableFocusWnd) && (msg.message == WM_KEYDOWN))
```
Comment (abhineetagarwal) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: Check here whether getEditableFocusWnd is not NULL

Comment (qjt376) [inline platform/wm/rhodes/browser/CEBrowserEngine.cpp]: Ok I will add that check

**Comment by abhineetagarwal** [comment]
Reviewed. You can merge the same...

**Comment by qjt376** [comment]
Thanks merging

**Pull request merged**